### DISCUSSION TITLE
Ability to Home 3d Chameleon

### DISCUSSION THREAD

**Discussion by challgren** [3DCoded/3dchameleon-klipper#6] [Ideas]
I noticed that there is no way to correctly home the 3d chameleon.

**Comment by 3DCoded** [comment]
The Chameleon does support homing. You can use `PULSE_CHAMELEON PULSES=7`. From the 3DChameleon GCode Generator:
<img width="834" alt="Screenshot 2024-04-13 at 9 08 04 AM" src="https://github.com/3DCoded/3dchameleon-klipper/assets/69043333/7f59cddd-6b50-4f66-8a7c-ed7c3ea4ce52">
